src/ontology/cob.Makefile

```diff
@@ -241,3 +248,60 @@ products/demo-cob.owl: $(TMPDIR)/demo-cob-merged.owl | products/
 
 $(TMPDIR)/subset-%.owl: $(TMPDIR)/merged-%.owl subsets/terms_%.txt | $(TMPDIR)
 	$(ROBOT) extract -m BOT -i $< -T subsets/terms_$*.txt -o $@
+
+# Crosscheck against upstream ontologies
+
+X_IMPORTS := BFO CHEBI CL DRON ENVO FOODON GO IAO MOP NCBITaxon OBI PATO PCO PO PR RO SO UBERON VO
+X_IMPORT_OWL_FILES := $(foreach n,$(X_IMPORTS), $(IMPORTDIR)/$(n)_import.owl)
+
+$(IMPORTDIR)/%_ontofox.txt: cob-edit.tsv
+	echo "[URI of the OWL(RDF/XML) output file]" > $@
+	echo "http://purl.obolibrary.org/obo/cob/dev/import/$*_import.owl" >> $@
+	echo "" >> $@
+	echo "[Source ontology]" >> $@
+	echo "$*" >> $@
+	echo "" >> $@
+	echo "[Low level source term URIs]" >> $@
+	grep "^$*:" $< | cut -f1 | sed "s!$*:!http://purl.obolibrary.org/obo/$*_!" >> $@
+	echo "" >> $@
+	echo "[Source annotation URIs]" >> $@
+	echo "http://www.w3.org/2000/01/rdf-schema#label" >> $@
+	echo "http://purl.obolibrary.org/obo/IAO_0000115" >> $@
+
+$(IMPORTDIR)/RO_ontofox.txt: cob-edit.tsv
+	echo "[URI of the OWL(RDF/XML) output file]" > $@
+	echo "http://purl.obolibrary.org/obo/cob/dev/import/RO_import.owl" >> $@
+	echo "" >> $@
+	echo "[Source ontology]" >> $@
+	echo "RO" >> $@
+	echo "" >> $@
+	echo "[Low level source term URIs]" >> $@
+	grep "^BFO:" $< | grep "owl:ObjectProperty" | cut -f1 | sed "s!BFO:!http://purl.obolibrary.org/obo/RO_!" >> $@
+	grep "^RO:" $< | cut -f1 | sed "s!RO:!http://purl.obolibrary.org/obo/RO_!" >> $@
+	echo "" >> $@
+	echo "[Source annotation URIs]" >> $@
+	echo "http://www.w3.org/2000/01/rdf-schema#label" >> $@
+	echo "http://purl.obolibrary.org/obo/IAO_0000115" >> $@
+
+$(IMPORTDIR)/%_import.owl: $(IMPORTDIR)/%_ontofox.txt
```

Comment:
Since you are not using the imports anywhere (other than the crosscheck), why checking it into version control?

Reply:
I want to remember the upstream annotations so I know when they change. I don't need the OntoFox config files, but they don't hurt. The cross check is important because it prevents the conflicts we need to solve in #293.

Reply:
Ok